main_keysync.go

```diff
@@ -85,10 +85,12 @@ func main() {
 		panic("invalid permissions specified")
 	}
 
-	var keyFileOwnerUID, keyFileOwnerGID int
-	n, err = fmt.Sscanf(inputFlags.keyFileOwnership, "%d:%d", &keyFileOwnerUID, &keyFileOwnerGID)
-	if err != nil || n != 2 {
-		panic("invalid ownership specified")
+	var keyFileOwnerUID, keyFileOwnerGID *int
+	if inputFlags.keyFileOwnership != "" {
+		n, err = fmt.Sscanf(inputFlags.keyFileOwnership, "%d:%d", keyFileOwnerUID, keyFileOwnerGID)
+		if err != nil || n != 2 {
+			panic("invalid ownership specified")
+		}
```

Comment:
Let's add a check here to make sure they are not negative.

Reply:
added